Gemfile.lock

```diff
@@ -1,57 +1,57 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (5.2.1)
-      actionview (= 5.2.1)
-      activesupport (= 5.2.1)
+    actionpack (5.2.2.1)
+      actionview (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       rack (~> 2.0)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.2)
-    actionview (5.2.1)
-      activesupport (= 5.2.1)
+    actionview (5.2.2.1)
+      activesupport (= 5.2.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.3)
-    activemodel (5.2.1)
-      activesupport (= 5.2.1)
-    activerecord (5.2.1)
-      activemodel (= 5.2.1)
-      activesupport (= 5.2.1)
+    activemodel (5.2.2.1)
+      activesupport (= 5.2.2.1)
+    activerecord (5.2.2.1)
+      activemodel (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       arel (>= 9.0)
-    activesupport (5.2.1)
+    activesupport (5.2.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
     arel (9.0.0)
     builder (3.2.3)
-    concurrent-ruby (1.1.1)
+    concurrent-ruby (1.1.5)
     crass (1.0.4)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.7.1)
-    i18n (1.1.1)
+    erubi (1.8.0)
```

Comment:
### [erubi](https://github.com/jeremyevans/erubi)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.7.1 → 1.8.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **11** commits. See the full changes on [the compare page](https://github.com/jeremyevans/erubi/compare/1.7.1...1.8.0).

These are the first 10 commits:
- (e98b24c) [Remove has_rdoc from gemspec, since it is deprecated](https://github.com/jeremyevans/erubi/commit/e98b24c1dcb986271818db269dea7565ef9091f4)
- (db21a52) [Update the README with an example of how to write a method t…](https://github.com/jeremyevans/erubi/commit/db21a52950d22c7df077f9808ad23dbd0cccae7f)
- (1d78fa0) [Add return_buffer option to CaptureEndEngine](https://github.com/jeremyevans/erubi/commit/1d78fa09658f2d61d2521b616898dc4b17f5c543)
- (5abd1c7) [Simplify test in attempt to get 1.8.7 passing](https://github.com/jeremyevans/erubi/commit/5abd1c76c30915c6d0fb6922676925b98b1c9ea4)
- (9761dea) [Modify spec to show how :return_buffer can be used when modi…](https://github.com/jeremyevans/erubi/commit/9761dea506d935bf3517f11e155d801f49e43aa8)
- (d822461) [Disable minitest plugins when testing](https://github.com/jeremyevans/erubi/commit/d82246113994a54aead526192d040b9e7095b957)
- (c3eac9f) [Flip `result` and `code` for :return_buffer option](https://github.com/jeremyevans/erubi/commit/c3eac9fac046c1e59032c9b77494799e3a25b81a)
- (d3bc72c) [Modify test to work with new :return_buffer behavior](https://github.com/jeremyevans/erubi/commit/d3bc72ce263e0c28d5d417b350ec626d294b1436)
- (f3939d1) [Rename return_buffer option to yield_returns_buffer](https://github.com/jeremyevans/erubi/commit/f3939d1680be34714248e5aba3180aa76dcc3230)
- (a60f764) [Fix and expand on documentation for :yield_returns_buffer](https://github.com/jeremyevans/erubi/commit/a60f764506c44b3df78c09cf0f278b44760122cc)


</details>